config/README.md

```diff
@@ -107,14 +108,15 @@ and makes all future kubecfg commands target this cluster.
 Follow the steps above for creating a new cockroachdb cluster, but with the
```

Comment:
CockroachDB (capitalization) -- let's make sure there capitalization is consistent throughout the document.